releases/release-1.26/release-notes/maps/pr-86139-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 86139
+releasenote:
+  text: <NONE>
```

Comment:
A new release note was added for this PR, see:
https://github.com/kubernetes/kubernetes/pull/86139#issue-536041701

Please add the new note from there to here.